ui/assets/css/hound.css

```diff
@@ -224,6 +224,9 @@ button:focus {
   color: #666;
   font-size: 24px;
   padding-bottom: 5px;
+  position: sticky;
+  top: 0;
+  background: #fff;
```

Comment:
There are some issues with the z-index of the repo title:
<img width="1029" alt="hound" src="https://user-images.githubusercontent.com/5206277/35299423-8eb020e0-0085-11e8-8684-e20063345f51.png">

I'd propose adding the following here:

```
padding: 1%;
z-index: 
```

`padding: 1%` is to make the title look a little natural.